zebra-state/src/service/finalized_state/disk_format/upgrade.rs

```
@@ -468,12 +472,7 @@ impl DbFormatChange {
         //   (unless a future upgrade breaks these format checks)
         // - re-opening the current version should be valid, regardless of whether the upgrade
         //   or new block code created the format (or any combination).
-        Self::format_validity_checks_detailed(db, cancel_receiver)?.unwrap_or_else(|_| {
-            panic!(
-                "unexpected invalid database format: delete and re-sync the database at '{:?}'",
-                db.path()
-            )
```

Why this change? (I haven't included it in the new PR, let me know if I should).

---

oops, leftover from debugging (this panic hides the original error... I guess the right fix would be to include the error in the panic message but you don't have to do that now)